eng/scripts/generatepatch.ps1

```diff
@@ -67,12 +66,22 @@ if (!$RemoteName) {
 }
 Write-Output "RemoteName is: $RemoteName"
 
+if (!$BranchName) {
+  $BranchName = GetBranchName -ArtifactId "generatepatch"
+}
```

Comment:
Powershell supports `??` if you want to use that:

```suggestion
$BranchName = $BranchName ?? (GetBranchName -ArtifactId "generatepatch")
```

Reply:
Wooow, thank you!